doc/lua_api.txt

```diff
@@ -9299,20 +9299,30 @@ Chat command definition
 
 Used by `minetest.register_chatcommand`.
 
+Specifies the function to be called and the privileges required when a player
+issues the command.  A help message that is the concatenation of the params and
+description fields is shown when the "/help" chatcommand is issued.
+
     {
-        params = "<name> <privilege>",  -- Short parameter description
+        params = "",
+		-- Short parameter description.  See the below note.
 
-        description = "Remove privilege from player",  -- Full description
+        description = "",
+		-- General description of the command's purpose.
 
-        privs = {privs=true},  -- Require the "privs" privilege to run
+        privs = {},
+		-- Required privileges to run. See [Privileges].
```

Comment:
The PR actually removes that mention there, as it was a fictitious example. 

It is mentioned again in the example section below, but if the use of a non-standard example string is objectionable, it could always be adapted.

Reply:
Maybe point to the example later in the section.

Reply:
A mention of `minetest.check_player_privs()` has been added.